fixed typos and extended doc

themataleao opened the pull request:
fixed typos and extended documentation

Bam4d reviewed (changes requested):
This is all good, but we are using single quotes in linting rules in the client. so single quotes in the Usage docs makes sense.

Also the `MistralClient` will find the API key in the environmental variables automatically. We could document this better also.

themataleao commented:
@Bam4d fixed the mentioned issues.

- fixed the issues with double quotes and added the single quote config to eslint rules.
- added doc for the automatic pull of env variable of the client
